apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -67,7 +67,7 @@ defmodule OMG.Watcher.DB.Transaction do
     DB.Repo.one(query)
   end
 
-  def get_by_filters(address, blknum, limit) do
+  def get_by_filters(address, blknum \\ nil, limit \\ nil) do
```

Comment:
Please revert - seem to be just for test, and I'm afraid of consequences

Reply:
Removing it, but I do feel having them optional makes the code clearer in terms of expectations (since it isn't handled until 2 functions down in pattern matching).